src/vs/platform/theme/common/colorRegistry.ts

```diff
@@ -183,6 +183,11 @@ export const inputValidationErrorBorder = registerColor('inputValidation.errorBo
 export const selectBackground = registerColor('dropdown.background', { dark: '#3C3C3C', light: Color.white, hc: Color.black }, nls.localize('dropdownBackground', "Dropdown background."));
 export const selectForeground = registerColor('dropdown.foreground', { dark: '#F0F0F0', light: null, hc: Color.white }, nls.localize('dropdownForeground', "Dropdown foreground."));
 export const selectBorder = registerColor('dropdown.border', { dark: selectBackground, light: '#CECECE', hc: contrastBorder }, nls.localize('dropdownBorder', "Dropdown border."));
+export const selectOptionCheckedForeground = registerColor('dropdown.optionCheckedForeground', { dark: Color.white, light: Color.white, hc: null }, nls.localize('dropdownOptionCheckedForground', "Dropdown option checked foreground."));
```

Comment:
@cleidigh actually thinking more about it, why do we have these colors at all? can we not just use the tree colors? since we are already using the same widget I see no much value in having different colors only for the dropdown....

Reply:
Using list colors.